FluentUI/src/main/res/values/strings.xml

```diff
@@ -193,10 +193,22 @@
     <!--Describes the state of a menu item that is not checked.-->
     <string name="popup_menu_accessibility_item_state_not_checked">not checked</string>
 
+    <!-- *** Calendar *** -->
+
+    <!--accessibility-->
```

Comment:
is description tag not required anywhere here for accurate translation ?

Reply:
Not used anywhere, but we can start using this or the. comment tag. What do. you suggest?